src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.ts

```
@@ -215,43 +219,45 @@ export class ProgrammingExerciseDetailComponent implements OnInit, OnDestroy {
+                        this.alertService.error(error.message);
+                    },
                 });
 
             this.exerciseStatisticsSubscription = this.statisticsService.getExerciseStatistics(exerciseId!).subscribe((statistics: ExerciseManagementStatisticsDto) => {
                 this.doughnutStats = statistics;
             });
+
+            if (this.programmingExercise.isAtLeastEditor) {
+                this.buildLogsSubscription = this.programmingExerciseService
+                    .getBuildLogStatistics(exerciseId!)
+                    .subscribe((buildLogStatistics) => (this.programmingExercise.buildLogStatistics = buildLogStatistics));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)